v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import CONST from '../../Constants';
+import LoadingCircle from '../../Shared/LoadingCircle';
+import PageTitle from '../../Shared/PageTitle';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+
+class PoliciesPage extends React.Component {
+    constructor(props) {
```

Comment:
I think we are now using redux to manage out state.  Or you and Kieran have agreed to let Kieran handle the changing of all the states to Redux?

Reply:
Waaat I didn't realize anything about redux lol @ki3ranoday can you handle this